src/Joomlatools/Console/Command/Site/Download.php

```diff
@@ -62,6 +69,12 @@ protected function configure()
                 InputOption::VALUE_REQUIRED,
                 'Alternative Git repository to clone. To use joomlatools/platform, use --repo=platform.'
             )
+	        ->addOption(
+	        	'clone',
+		        null,
+		        InputOption::VALUE_REQUIRED,
```

Comment:
@msaladna I would like to suggest we use `VALUE_NONE` here and turn it into a simple boolean flag. (see [docs](https://symfony.com/doc/current/console/input.html#using-command-options))

What if we rename the `clone` option to `download` and use default behavior (cloning) if the flag is unset, and download directly if it is present? Since technically the `clone` option shouldn't require a value.

Reply:
Done. --clone is now --download and requires an additional --repo argument. Otherwise with its default invocation joomlatools will rely on --repo=platform and download tarballs from github.